collect_app/src/androidTest/assets/forms/simple-search-external-csv.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's too bad to have duplicates of these test files in `androidTest` and `test`. I think it's the right thing to do these Robolectric tests with real form files, though.

This might actually be a good reason for all tests to live in the same source set, as the androidx libs want.

Reply:
I think it's fine to duplicate. A nice solution is to bundle the test forms in gradle submodule if it gets annoying. I have thoughts on everything living in the same set but they can be a fun discussion another time!